addons/missionModules/functions/fnc_zeus_banziCharge.sqf

```diff
@@ -0,0 +1,15 @@
+#include "script_component.hpp"
+
+params ["_logic"];
+
+if (!local _logic) exitWith {};
+private _unit = _logic getVariable ["bis_fnc_curatorAttachObject_object",objnull];
```

Comment:
Do we want to use the dismount workaround logic here? `bis_fnc_curatorAttachObject_object` should be accurate, but may not be....